test/unit/resources/postgres_session_test.rb

```diff
@@ -0,0 +1,26 @@
+# encoding: utf-8
+# author: Aaron Lippold
+
+require 'helper'
+
+describe 'Inspec::Resources::PostgresSession' do
+  it 'verify postgres_session basic init configuration' do
```

Comment:
This isn't something I would test. Instead, validate that the command you're going to hand off to `inspec.command` is what you believe it should be. This will eliminate the need to expose the user/pass/host as attr_readers; you don't need those exposed to the user since they are the one that supplied them.